### PR TITLE
ReusableAssets app integration

### PR DESCRIPTION
**Pull request by MarcelStraka**
Base folder for reusables

### CODE REVIEW

**Comment by MarcelStraka** [comment]
Base folder for reusables

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (AlexanderHodicke) [inline .vscode/settings.json:1]: @oheimburger whats this file? To be removed?

Comment (MarcelStraka) [reply .vscode/settings.json:1]: No idea how it got there. Maybe somehow from VS Code? We can remove it

File: application-development/application-integration-and-process-automation/reusable-assets-cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Reusable Assets: Cloud
```
Comment (AlexanderHodicke) [inline application-development/application-integration-and-process-automation/reusable-assets-cloud/README.md:1]: @MarcelStraka This is a custom page under app integration. 

I suggested having here the product folders and their folders.

Comment (MarcelStraka) [reply application-development/application-integration-and-process-automation/reusable-assets-cloud/README.md:1]: These assets relate to several cloud services, this is why we have it under the root. We have the same in Confluence

**Review by AlexanderHodicke — APPROVED** [review]

**Pull request merged**